src/sentry/search/events/constants.py

```diff
@@ -1,4 +1,7 @@
 import re
+from typing import Dict
+
+from typing_extensions import TypedDict
```

Comment:
Is this different than `from typing import TypedDict`?

Reply:
Ah woops, this is me doing copy paste. Should be importing from typing here will update